searx/engines/google.py

```diff
@@ -331,17 +376,21 @@ def response(resp):
     dom = html.fromstring(resp.text)
     data_image_map = _parse_data_images(dom)
 
-    # results --> answer
-    answer_list = eval_xpath(dom, '//div[contains(@class, "LGOjhe")]')
-    for item in answer_list:
-        for bubble in eval_xpath(item, './/div[@class="nnFGuf"]'):
-            bubble.drop_tree()
-        results.append(
-            {
-                'answer': extract_text(item),
-                'url': (eval_xpath(item, '../..//a/@href') + [None])[0],
-            }
-        )
+    results = _eval_answers(results, dom, '//div[contains(@class, "card-section")]')  # Look for cards first
+    if not results:
+        results = _eval_answers(results, dom, '//div[contains(@class, "LGOjhe")]')  # Look for rendered answers next
+    # Look for JS DOM encoded string answers last
```

Comment:
Do you have an example query at hand, that "Look for JS DOM encoded string answers"?

From my tests:

     is vanilla antibacterial              --> # Look for rendered answers
     what is the capital of england        --> # Look for rendered answers
     what time is it in the united states  --> # Look for cards
     what is 3+17                          --> # Look for cards

Reply:
Interesting. For me, `what is the capital of england` comes up in the JS DOM.

If you look at it in inspector in the browser, it looks like it's in the normal DOM, but that's because the DOM was generated via JS. If you look at it in the result that `searchngx` receives, it's in the JS.

Reply:
> If you look at it in inspector in the browser, it looks like it's in the normal DOM, but that's because the DOM was generated via JS. If you look at it in the result that searchngx receives, it's in the JS.

Nope, I dump the HTML response from the google engine into a file and open this file in a HTML editor (there is no JS active in this procedure).